src/Pages/Main/Component/MainAll/MainSectionRecommend/MainSectionRecommend.js

```diff
@@ -17,33 +19,43 @@ class MainSectionRecommend extends Component {
           recommendList: data.results,
         });
       });
-    // console.log('data: ', this.state.recommendList);
   }
 
+  recommendCarousel = () => {};
+
   render() {
     const { recommendList } = this.state;
-    // console.log('data: ', this.state.recommendList);
     return (
       <section className="MainSectionRecommend">
-        <div className="MainSectionRecommendWrapper">
-          <h2>나만 알고싶은 향기</h2>
-          <ul>
-            {recommendList &&
-              recommendList.map(recommend => {
-                return (
-                  <li key={recommend.id}>
-                    <img
-                      src={recommend.image_url}
-                      alt="나만 알고싶은 향기 이미지"
-                    />
-                    <h3>{recommend.name}</h3>
-                    {/* <p>{recommend.name}</p> */}
-                    <p>{recommend.description}</p>
-                    <p>{recommend.price} 원</p>
-                  </li>
-                );
-              })}
-          </ul>
+        <div className="recommendWrapper">
+          <button className="backButton">
+            <i class="fas fa-caret-left"></i>
+          </button>
+          <div className="recommendContainer">
+            <h2>4월에 어울리는 향기</h2>
+            <div className="recommendCarousel">
+              {recommendList &&
+                recommendList.map((recommend, idx) => {
```

Comment:
```suggestion
              {recommendList.map((recommend, idx) => {
```
`[]`은 언제나 true이기 때문에 작성해주신 조건식은 언제나 true 경우만을 고려하는 조건식이 됩니다. 불필요하므로 제거해주세요.